### PR TITLE
Check for Tumbleweed when it boots on s390x

### PR DESCRIPTION
**Pull request by SergioAtSUSE**
- Related ticket: https://progress.opensuse.org/issues/63349
- Verification run: http://openqa.slindomansilla-vm.qa.suse.de/tests/2193

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -1327,7 +1327,10 @@ sub reconnect_mgmt_console {
     $args{timeout} //= 300;
 
     if (check_var('ARCH', 's390x')) {
-        my $login_ready = qr/Welcome to SUSE Linux Enterprise Server.*\(s390x\)/;
+        my $login_ready = qr/Welcome to openSUSE Tumbleweed/;
+        if (is_sle()) {
+            $login_ready = qr/Welcome to SUSE Linux Enterprise Server.*\(s390x\)/;
+        }
```
Comment (okurz) [inline lib/utils.pm]: how about

```perl
my $login_ready = is_sle ? qr/Welcome to SUSE Linux Enterprise Server.*\(s390x\)/ : qr/Welcome to openSUSE Tumbleweed/;
```

if you think this is getting too long:

```perl
my $login_ready = qr/Welcome to /;
$login_ready .= is_sle ? qr/SUSE Linux Enterprise .*\(s390x\)/ : qr/openSUSE Tumbleweed/;
```

**Comment by SergioAtSUSE** [comment]
Done, new verification run: http://openqa.slindomansilla-vm.qa.suse.de/tests/2194

File: lib/utils.pm [hunk 2]
```diff
@@ -1327,7 +1327,10 @@ sub reconnect_mgmt_console {
     $args{timeout} //= 300;
 
     if (check_var('ARCH', 's390x')) {
-        my $login_ready = qr/Welcome to SUSE Linux Enterprise Server.*\(s390x\)/;
+        my $login_ready = qr/Welcome to /;
+        if (is_sle()) {
```
Comment (okurz) [inline lib/utils.pm]: this is unfortunately looks wrong now. I doubt you need this if at all

**Comment by SergioAtSUSE** [comment]
New verification run: http://openqa.slindomansilla-vm.qa.suse.de/tests/2195

**Pull request merged**